src/Components/Ennumerated_Components/Book.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
Job well-done @SotoArmando, 👍 
According to the previous review, you were supposed to move this to the container folder right?

**I quote from @MauricioRobayo comments** 
`One of the main purposes of this milestone is learning about presentational components and container components.`

`Container components will manage the state (in this case the ones that will connect to the Redux store). Presentational components are stateless`

**In other words;** In React file structure, any component that is connecting to the redux store should be in the `container folder`.

[Helpful Article](https://krasimirtsonev.com/blog/article/react-js-presentational-container-components)